test/unit/infinite-scroll.ts

```diff
@@ -5,6 +5,13 @@ import suite from './_suite';
 
 const STRUCTURE = { a: 'b' };
 
+interface Global {
+  window: any,
+}
+declare var global: Global;
+
+global.window = {};
```

Comment:
Sorry if I was unclear. Instead of injecting a `window` object into the global scope, you should just stub the `WINDOW` function to return a fake.

Example: https://github.com/groupby/storefront-sayt/blob/master/test/unit/sayt.ts#L53